### PR TITLE
New colors

### PR DESCRIPTION
**Pull request by qwhex**
screen shot 2018-05-06 at 11 27 20

### CODE REVIEW

File: dist/colornames.csv [hunk 1]
```diff
@@ -4016,6 +4018,7 @@ Dark Highlight,#705446
 Dark Horizon,#666699
 Dark Hot Pink,#d90166
 Dark Imperial Blue,#00416a
+Dark Incremental Blue,#123456
```
Comment (meodai) [inline dist/colornames.csv]: Can we  simply call this Incremental Blue?

File: dist/colornames.csv [hunk 2]
```diff
@@ -8653,6 +8658,7 @@ Light Iced Lavender,#d0d4e3
 Light Image Tone,#ccd0da
 Light Imagine,#aed4d8
 Light In The Slip,#ead1da
+Light Incremental Blue,#456789
```
Comment (meodai) [inline dist/colornames.csv]: "A hint of Increment" maybe?

File: dist/colornames.csv [hunk 3]
```diff
@@ -11645,6 +11652,7 @@ Phosphor Green,#00aa00
 Phosphorus,#a5d0c6
 Photo Grey,#aead96
 Photon White,#f8f8e8
+PHP,#8892bf
```
Comment (meodai) [inline dist/colornames.csv]: Would be nice to add the name of the color name like. PHP Purple or something

**Comment by meodai** [comment]
If you have more ideas check #38

**Comment by qwhex** [comment]
I wrote a script to find colors which are the most distant from existing named colors. It needs a little bit more polishing, but then I'd gladly contribute it to the repository.

I will make the changes you requested when I get home. I hope you got the joke about #123456 and "Incremental Blue".

**Comment by meodai** [comment]
@qwhex very curious about the script! Yes i got the 123456 pun :D

**Comment by qwhex** [comment]
I made the requested changes.
Yesterday I saw your visualization on reddit, could you send a link to the source code? I'd like to combine it with my gap-finder functions.

**Comment by meodai** [comment]
I'm sorry I messed up the merge, maybe its better if we only commit the SRC file in the future. I manually added them in c331f20f00a17fa15cd5c53716db489a158429aa I will add you to the contributors. Sorry for that

**Pull request closed without merging**